posthog/models/early_access_feature.py

```diff
@@ -30,6 +30,7 @@ class Stage(models.TextChoices):
     stage = models.CharField(max_length=40, choices=Stage.choices)
     documentation_url = models.URLField(max_length=800, blank=True)
     created_at = models.DateTimeField(auto_now_add=True)
+    feature_management = models.ForeignKey("posthog.Feature", on_delete=models.SET_NULL, null=True)
```

Comment:
"Why `feature_management` vs just `feature`?" 

The existing `feature_flag` foreign key on some of these models uses related names that would clash with `feature`:
```
related_name="features",
related_query_name="feature",
```

Reply:
ah, that's a bit annoying, but I understand the choice.